src/ports/postgres/modules/deep_learning/madlib_keras_validator.py_in

```diff
@@ -53,15 +178,15 @@ class FitInputValidator:
 
     def _validate_input_table(self, table):
         _assert(is_var_valid(table, self.independent_varname),
-                "model_keras error: invalid independent_varname "
-                "('{independent_varname}') for table "
+                "model_keras error: invalid independent_varname "\
```

Comment:
Do we need these `\` characters? The string does not have the new line, thanks to the way python string concatenation works.

Reply:
Sure, will change it, I don't feel strongly either way.
I put it in there since I thought it was clearer to read the code. `\` indicates that I am splitting a single error message across multiple lines in the code (to limit the number of chars showing up in a single line in the python file), although it's still a single line as an error message itself. Python does that implicitly as you pointed out, but I put it there to show it up explicitly in the code.